rpc/src/rpc.rs

```diff
@@ -7354,6 +7452,293 @@ pub mod tests {
+                assert!(*sizes.get(&RpcAccountIndex::SplTokenOwner).unwrap() == 2);
+                // 3) Mint1 - Is in 2 SPL Accounts
+                let req = format!(
+                    r#"{{"jsonrpc":"2.0","id":1,"method":"getSecondaryIndexKeySize","params":["{}"]}}"#,
+                    mint1_pubkey,
+                );
+                let res = io.handle_request_sync(&req, meta.clone());
+                let result: Value = serde_json::from_str(&res.expect("actual response"))
+                    .expect("actual response deserialization");
+                let sizes: HashMap<RpcAccountIndex, usize> =
+                    serde_json::from_value(result["result"]["value"].clone()).unwrap();
+                assert!(*sizes.get(&RpcAccountIndex::SplTokenMint).unwrap() == 2);
+                // 4) Mint2 - Is in 1 SPL Account
+                let req = format!(
+                    r#"{{"jsonrpc":"2.0","id":1,"method":"getSecondaryIndexKeySize","params":["{}"]}}"#,
+                    mint2_pubkey,
+                );
+                let res = io.handle_request_sync(&req, meta.clone());
+                let result: Value = serde_json::from_str(&res.expect("actual response"))
+                    .expect("actual response deserialization");
+                let sizes: HashMap<RpcAccountIndex, usize> =
+                    serde_json::from_value(result["result"]["value"].clone()).unwrap();
+                assert!(*sizes.get(&RpcAccountIndex::SplTokenMint).unwrap() == 1);
+                // 5) SPL Token Program Owns 6 Accounts
+                let req = format!(
+                    r#"{{"jsonrpc":"2.0","id":1,"method":"getSecondaryIndexKeySize","params":["{}"]}}"#,
+                    inline_spl_token::id(),
+                );
+                let res = io.handle_request_sync(&req, meta.clone());
+                let result: Value = serde_json::from_str(&res.expect("actual response"))
+                    .expect("actual response deserialization");
+                let sizes: HashMap<RpcAccountIndex, usize> =
+                    serde_json::from_value(result["result"]["value"].clone()).unwrap();
+                assert!(*sizes.get(&RpcAccountIndex::ProgramId).unwrap() == 6);
+                // 5) System Program Owns 4 Accounts: [Wallet1, Wallet2, SPLTokenProgram]
+                let req = format!(
+                    r#"{{"jsonrpc":"2.0","id":1,"method":"getSecondaryIndexKeySize","params":["{}"]}}"#,
+                    system_program::id(),
+                );
+                let res = io.handle_request_sync(&req, meta.clone());
+                let result: Value = serde_json::from_str(&res.expect("actual response"))
+                    .expect("actual response deserialization");
+                let sizes: HashMap<RpcAccountIndex, usize> =
+                    serde_json::from_value(result["result"]["value"].clone()).unwrap();
+                assert!(*sizes.get(&RpcAccountIndex::ProgramId).unwrap() == 4);
```

Comment:
Same comments about before/after testing or at least commenting about this number

Reply:
Added an initial query as suggested.